docs/faq/pingfederate.md

```diff
@@ -0,0 +1,9 @@
+# Frequently Asked Questions - PingFederate
+
+## What versions of PingFederate are supported?
+
+The PingFederate provider supports versions `11.2` through `12.0` of PingFederate.
```

Comment:
Might be better to link the registry page so that we don't have to worry about keeping this doc page in sync